vendor/opam-0install/lib/model.ml

```diff
@@ -193,7 +192,8 @@ module Make (Monad : S.Monad) (Context : S.CONTEXT with type 'a monad = 'a Monad
                   |> list_deps ~context ~importance
                 in
                 make_deps `Essential ensure OpamFile.OPAM.depends @
-                make_deps `Restricts prevent OpamFile.OPAM.conflicts
+                make_deps `Restricts prevent OpamFile.OPAM.conflicts @
+                make_deps `Recommended ensure OpamFile.OPAM.depopts
```

Comment:
@kit-ty-kate is it ok to support depopts like this?

Reply:
I don’t think so. That sounds like a change in semantics from what opam does. See the note in https://opam.ocaml.org/doc/Manual.html#opamfield-depopts

Reply:
I find the following line hard to interpret in the case of dune:

> It contains packages that will be used, if present, by the package being defined, either during build or runtime, but that are not required for its installation

The only packages that are present when building the packages that are only those requested by the user (e.g. the `depends` field). So there's no way for a depopt to be opportunistically present this way.

Reply:
mmh, thinking about it again, the change of semantics should be fine for dune actually.